src/ROCrate/ROCrateObject.fs

```diff
@@ -39,4 +39,19 @@ type ROCrateObject(id:string, schemaType: string, ?additionalType) =
 
         member this.AdditionalType
             with get() = _additionalType
-            and set(value) = _additionalType <- value
\ No newline at end of file
+            and set(value) = _additionalType <- value
+
+    member this.SetContext (context: #DynamicObj) =
```

Comment:
I would argue for creating an "LDContext" or "Context" type which we can reference here.

Reply:
For now it can just be an otherwise empty class inheriting DynamicObj

Reply:
Because we will need special behaviour for these objects sometime in the future and it's probably easier if the alias class is already there and referenced where it's used

Reply:
Not the biggest fan of adding stubs/unused code/types but will do